extensions/src/poseHand/index.ts

```diff
@@ -0,0 +1,324 @@
+import { ArgumentType, BlockType, Extension, Block, DefineBlock, Environment, ExtensionMenuDisplayDetails, RuntimeEvent } from "$common";
+
+import Video from '../../../packages/scratch-vm/src/io/video';
+import * as handpose from '../../../packages/scratch-vm/node_modules/@tensorflow-models/handpose/dist';
```

Comment:
We'll want to install this package as dependency to your extension.

To do so:

1. Change directories (`cd`) into your extension folder 
```bash
cd extensions/src/poseHand
```
2. Install this package: 
```bash
npm install @tensorflow-models/handpose
```
3. Import using: 
```
import * as handpose from '@tensorflow-models/handpose';
```
Or you might be able to even just do `import handpose from '@tensorflow-models/handpose'`
4. Commit changes to package files